ui/apps/platform/src/Containers/NetworkGraph/NetworkGraphContainer.tsx

```diff
@@ -222,6 +222,16 @@ function getDisplayEdges(edges: CustomEdgeModel[], showEdgeLabels: boolean): Cus
     });
 }
 
+function removeDNSFlows(edges: CustomEdgeModel[]) {
+    return edges.filter((edge) => {
+        return (
+            edge?.data?.sourceToTargetProperties[0]?.port !== 53 &&
+            edge?.data?.sourceToTargetProperties[0]?.port !== 5353 &&
+            edge?.data?.sourceToTargetProperties[0]?.protocol !== 'L4_PROTOCOL_UDP'
```

Comment:
Is it possible that the `sourceToTargetProperties` array ever has more than one item, at index 0 ?

Reply:
That's a good question. Now that I looked at this again, we also have a `targetToSourceProperties` field that only appears when it's a bidirectional edge. So I should double-check that there's no issue with that either. I'll look at this logic again